src/System.Text.Json/src/System/Text/Json/Node/JsonObject.cs

```diff
@@ -534,14 +369,14 @@ public bool TryGetJsonArrayPropertyValue(string propertyName, out JsonArray json
         /// <summary>
         ///  A collection containing the property values of JSON object.
         /// </summary>
-        public ICollection<JsonNode> PropertyValues => _dictionary.Values;
+        public ICollection<JsonNode> PropertyValues => _dictionary.Values.Select(jsonObjectProperty => jsonObjectProperty._value).ToList();
```

Comment:
Oh, I missed this earlier.  We usually don't let properties return interfaced collections, unless they got their collection as an interfaced ctor parameter (or are settable).

This should possibly say List; but it's expensive, so it shouldn't be a property.  (Dictionary got this wrong).

Reply:
Ok, I see, I'll change it to method.